src/GROWdb_variable_puller.Rmd

```diff
@@ -28,7 +30,7 @@ sf_use_s2(FALSE)
 # url = paste0("https://edap-ow-data-commons.s3.amazonaws.com/NHDPlusV21/", "Data/NationalData/NHDPlusV21_NationalData_Seamless", "_Geodatabase_Lower48_07.7z"),
 # progress = TRUE)
 
-all_catch <- sf::st_read(dsn = "data/in/NHDPlusV21_NationalData_Seamless", "_Geodatabase_Lower48_07.gdb",
+all_catch <- st_read(dsn = "data/in/NHDPlusV21_NationalData_Seamless", "_Geodatabase_Lower48_07.gdb",
```

Comment:
nitpick: same in this script as the app script - these functions aren't masked by other packages, so you don't need the prefix.